docs/getting-started-guides/docker-multinode.md

```diff
@@ -74,10 +74,11 @@ it is still useful to use containers for deployment and management, so we create
 You can specify the version on every node before install:
 
 ```sh
-export K8S_VERSION=<your_k8s_version (e.g. 1.2.0-alpha.6)>
+export K8S_VERSION=<your_k8s_version (e.g. 1.2.0-alpha.7)>
```

Comment:
Good question. I checked that `curl -sSL https://storage.googleapis.com/kubernetes-release/release/latest.txt` works as expected. I'll to find out why `console` link requires authentication.